resources/views/blade/input/textarea.blade.php

```diff
@@ -0,0 +1,11 @@
+@props([
+    'value' => '',
+    'cols' => 50,
```

Comment:
I hadn't noticed this, but we don't actually use cols, since we let bootstrap handle the width.

Reply:
The `Form::textarea` call actually added those in there so the resulting html included it. Should I remove it as an prop in the component?

Reply:
Probably - it won't matter, since the CSS will override it, but we don't need it. Not urgent tho. The behavior will be the same.

Reply:
Just did it so I didn't forget about it 😅 

https://github.com/snipe/snipe-it/pull/16225